Makefile

```diff
@@ -40,6 +40,6 @@ start:
 build: check-env clone
 	@ printf "\n[${GREEN} Generating environment variables... ${NO_COLOR}]\n"
 	cd ${SDX_HOME}/sdx-common && pip install .; cd -
-	cd ${SDX_HOME}/sdx-ops && ${PYTHON3} sdx/ops/configure.py --env > ${SDX_HOME}/sdx-compose/env/private.env ;cd -
+	cd ${SDX_HOME}/sdx-ops && python sdx/ops/configure.py --env > ${SDX_HOME}/sdx-compose/env/private.env ;cd -
```

Comment:
I actually find it really annoy you need a virtual environment to run a docker-compose script, it shouldn't even be using Python, all it's doing is building and starting docker containers.

I agree with James, why are we even generating keys? Why not just use the test keys? We could remove all that and keep the script much simpler.

Though if we're determined to create keys just convert the configure.py script to a bash script, it's basically 4 openssl commands. Then we don't have to create a virtual environments to install sdx-common and we probably don't even need Python on the path at all.  

Either of these options simplifies a Makefile that at the minute doesn't work out of the box.